bootstrap/bootstrap.go

```diff
@@ -91,7 +128,17 @@ type bootstrap struct {
 	managers *managers.Managers
 }
 
-func init() {
+// Config returns the server configuration
+func (bs *bootstrap) Config() *config.Config {
+	return bs.cfg
+}
+
+// Managers returns bootstrapped identity-managers
+func (bs *bootstrap) Managers() *managers.Managers {
+	return bs.managers
+}
+
+func Boot(ctx context.Context, bsConf *Config, serverConf *config.Config) (Bootstrap, error) {
```

Comment:
Can you please add a comment for this function. It is the main entry point of the package and imo deserves some :)